src/extension.ts

```diff
@@ -53,6 +54,8 @@ export function activate(context: vscode.ExtensionContext) {
 
   setGlobalContext();
 
+  new ContextExplorer(context);
```

Comment:
If we determine the view name, it would be better to change this class and file name.